src/renderer/components/ft-input-tags/ft-input-tags.vue

```diff
@@ -20,7 +20,7 @@
           v-for="tag in tagList"
           :key="tag.id"
         >
-          <label>{{ tag }}</label>
+          <span>{{ tag }}</span>
           <font-awesome-icon
             :icon="['fas', 'fa-times']"
```

Comment:
We should probably also add a ':title' and an ":aria-label' to this element that will be "Remove {tag}" so people with screen readers can tell that this is a remove button

Reply:
@ChunkyProgrammer  et al,

Do you mind if I do the screenreader accessibility update in a separate PR. This one atm fixes the issue its supposed to and It looks like it will take me a while to sort a screenreader fix.

Currently struggling to get a screenreader to work with freetube at all atm. I've got it reading out normal OS menu's etc but as soon as I tab to freetube it stops saying anything - same thing happens with brave, but some programs work like my password safe.

for reference i'm using Orca with espeak ( the built in linux screenreader with gnome )

Reply:
@petaded I understand you wanting to test accessibility changes, however as adding the title and aria-label attributes is standard web accessibility stuff, I think it's fine to do it without testing it with a screen reader.